graphql_compiler/scripts/generate_test_sql/__init__.py

```diff
@@ -1,14 +1,38 @@
-# Copyright 2017-present Kensho Technologies, LLC.
+# Copyright 2018-present Kensho Technologies, LLC.
+import codecs
 import datetime
 import os
 import random
+import re
 import sys
 
-from ... import __version__
 from .animals import get_animal_generation_commands
 from .species import get_species_generation_commands
 
 
+#  https://packaging.python.org/guides/single-sourcing-package-version/
+#  #single-sourcing-the-version
+
+
+def read_file(filename):
+    """Read package file as text to get its version"""
```

Comment:
there's no version checking going on in this function

Reply:
Didn't quite get this comment.

Reply:
`read_file` has a docstring that says it gets a version -- no such thing happens in this function, and what the caller uses this function for is the caller's business. I think the docstring should be updated.